src/content-helper/editor-sidebar/title-suggestions/component.tsx

```diff
@@ -84,8 +112,23 @@ export const TitleSuggestionsPanel = (): JSX.Element => {
 				request_more: titles.length > 0,
 				total_titles: titles.length,
 				total_pinned: titles.filter( ( title ) => title.isPinned ).length,
+				tone: tone ? tone : 'neutral',
+				persona: persona ? persona : 'journalist',
```

Comment:
@acicovic we don't, since those basically work as the key for the `{ label, emoji }` object.  We might be able to expose the default by exporting it from the component, I'm going to take a look and see if I can implement that :)